packages/fxa-auth-server/bin/subhub_messaging.js

```diff
@@ -5,25 +5,51 @@
 
 'use strict';
 
+const LIB_DIR = '../lib';
+
 // This MUST be the first require in the program.
 // Only `require()` the newrelic module if explicity enabled.
 // If required, modules will be instrumented.
-require('../lib/newrelic')();
+require(`${LIB_DIR}/newrelic`)();
 
 const config = require('../config').getProperties();
-const log = require('../lib/log')(config.log.level, 'subhub-messaging');
-const Token = require('../lib/tokens')(log, config);
-const SQSReceiver = require('../lib/sqs')(log);
-// FIXME: import a different module:
-const subhubUpdates = require('../lib/subhub/updates')(log, config);
-
-const DB = require('../lib/db')(config, log, Token);
-
-const subhubUpdatesQueue = new SQSReceiver(
-  config.subhubServerMessaging.region,
-  [config.subhubServerMessaging.subhubUpdatesQueueUrl]
-);
-
-DB.connect(config[config.db.backend]).then(db => {
```

Comment:
I moved this stuff into an `async` function so that we could `await`. Apologies for the diff noise, the substantive change is to initialise the mailer and pass it to subhub.